### PR TITLE
[Improvement][MER-2087] Add pagination to add materials modal

### PR DESCRIPTION
**Pull request by augusto1024**
[Link to the ticket](https://eliterate.atlassian.net/browse/MER-2087)

### CODE REVIEW

File: lib/oli_web/live/common/hierarchy/hierarchy_picker.ex [hunk 1]
```diff
@@ -121,15 +124,34 @@ defmodule OliWeb.Common.Hierarchy.HierarchyPicker do
...
+        <%!-- <%= for pub <- @publications do %>
 
           <div id={"hierarchy_item_#{pub.id}"}>
             <button class="btn btn-link ml-1 mr-1 entry-title" phx-click="HierarchyPicker.select_publication" phx-value-id={pub.id}>
               <%= pub.project.title %>
             </button>
           </div>
 
-        <% end %>
+        <% end %> --%>
```
Comment (eliknebel) [inline lib/oli_web/live/common/hierarchy/hierarchy_picker.ex:154]: Is this `--%>` leftover from an old comment block?

File: lib/oli_web/live/delivery/remix_section.ex [hunk 2]
```diff
@@ -752,6 +823,27 @@ defmodule OliWeb.Delivery.RemixSection do
...
+          Enum.filter(
+            publications,
+            &(&1.project.title
+              |> String.downcase()
+              |> String.contains?(String.downcase(text_filter)))
+          )
+      end
+
+    total_count = length(filtered_publications)
+    publications = Enum.slice(filtered_publications, params[:offset], params[:limit])
```
Comment (simonchoxx) [inline lib/oli_web/live/delivery/remix_section.ex:842]: Great use of `Enum.slice()` here 👏

**Review by simonchoxx — COMMENTED** [review]
LGTM!!! 🚀🚀

File: lib/oli_web/live/common/hierarchy/hierarchy_picker.ex (hunk 1 above)
Comment (augusto1024) [reply lib/oli_web/live/common/hierarchy/hierarchy_picker.ex:154]: Oops

**Review by eliknebel — APPROVED** [review]

**Pull request merged**